README.md

```diff
@@ -333,7 +351,7 @@ to be notified when the action completes:
 * `useLightTheme([callback])` - sets the editor to use the light theme (default)
 * `useDarkTheme([callback])` - sets the editor to use the dark theme
 * `showSidebar([callback])` - opens the file tree sidebar
-`* `hideSidebar([callback])` - hides the file tree sidebar
+  `* `hideSidebar([callback])` - hides the file tree sidebar
```

Comment:
Not sure what this change is? It doesn't actually indent the list item (when you look at the rich diff). Also, one error that was already in the markdown here is the extra \` before the asterisk. That can probably be fixed.